src/main/java/org/ibp/api/rest/dataset/DatasetResource.java

```diff
@@ -93,7 +93,7 @@ public class DatasetResource {
 
 	@ApiOperation(value = "Get Dataset Columns", notes = "Retrieves ALL MeasurementVariables (columns) associated to the dataset, "
 		+ "that will be shown in the Observation Table")
-	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES', 'MANAGE_STUDIES', 'BROWSE_STUDIES')" + PermissionsEnum.HAS_MANAGE_STUDIES_VIEW)
+	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES', 'MANAGE_STUDIES', 'VIEW_STUDIES')" + PermissionsEnum.HAS_MANAGE_STUDIES_VIEW)
```

Comment:
@cuenyad this comment applies to all this class
should we add VIEW_STUDIES to HAS_MANAGE_STUDIES_VIEW and simplify most of the PreAuthorities in this resource? 
Not that view studies is below manage studies, makes sense to me to add it to the enum and simplify the list or authorities when this is used:
``
hasAnyAuthority('ADMIN','STUDIES', 'MANAGE_STUDIES', 'VIEW_STUDIES')" + PermissionsEnum.HAS_MANAGE_STUDIES_VIEW
``

Reply:
For now, I prefer to keep View Studies separate from HAS_MANAGE_STUDIES_VIEW, to maintain the same permissions as defined in MANAGE_STUDIES_VIEW_PERMISSIONS from permissions.ts. When we start with the granular permissions in the study, the HAS_MANAGE_STUDIES_VIEW will not make sense anymore.